src/ILLink.RoslynAnalyzer/RequiresUnreferencedCodeAnalyzer.cs

```diff
@@ -72,35 +73,16 @@ public sealed class RequiresUnreferencedCodeAnalyzer : RequiresAnalyzerBase
 		protected override bool IsAnalyzerEnabled (AnalyzerOptions options, Compilation compilation) =>
 			options.IsMSBuildPropertyValueTrue (MSBuildPropertyOptionNames.EnableTrimAnalyzer, compilation);
 
-		protected override ImmutableArray<ISymbol> GetSpecialIncompatibleMembers (Compilation compilation)
-		{
-			var incompatibleMembers = ImmutableArray.CreateBuilder<ISymbol> ();
-			var typeType = compilation.GetTypeByMetadataName ("System.Type");
-			if (typeType != null) {
-				incompatibleMembers.AddRange (typeType.GetMembers ("MakeGenericType").OfType<IMethodSymbol> ());
-			}
-
-			var methodInfoType = compilation.GetTypeByMetadataName ("System.Reflection.MethodInfo");
-			if (methodInfoType != null) {
-				incompatibleMembers.AddRange (methodInfoType.GetMembers ("MakeGenericMethod").OfType<IMethodSymbol> ());
-			}
-
-			var expressionCallType = compilation.GetTypeByMetadataName ("System.Linq.Expressions.Expression");
-			if (expressionCallType != null) {
-				incompatibleMembers.AddRange (expressionCallType.GetMembers ("Call").OfType<IMethodSymbol> ()
-					.Where (m => m.Parameters.Length == 4 && m.Parameters[0] is IParameterSymbol parameterSymbol
-					&& parameterSymbol.Type?.ContainingNamespace?.Name == "System" && parameterSymbol.Type?.Name == "Type"));
-			}
-
-			return incompatibleMembers.ToImmutable ();
-		}
+		/// <summary>
+		/// Add methods that the Trimmer handles intrinsically to an exlusionary list
+		/// </summary>
+		/// <param name="compilation"></param>
+		/// <returns></returns>
```

Comment:
nit: left-over comment for the removed method

Reply:
Thanks for catching this!